src/nvim/testdir/test_window_cmd.vim

```diff
@@ -924,4 +924,18 @@ func Test_window_resize()
   %bwipe!
 endfunc
 
+" Test for errors with :wincmd
+func Test_wincmd_errors()
+  call assert_fails('wincmd g', 'E474:')
+  call assert_fails('wincmd ab', 'E474:')
+endfunc
+
+" Test for errors with :winpos
+func Test_winpos_errors()
+  if !has("gui_running") && !has('win32')
+    call assert_fails('winpos', 'E188:')
+  endif
+  call assert_fails('winpos 10', 'E466:')
+endfunc
```

Comment:
Is `winpos` purposefully not included in nvim or it just wasn't ported?

Reply:
https://github.com/neovim/neovim/commit/e0e41b30#diff-b22503b06d373fd31e6496ab2791935466d123f9b69fa2c62a89d9cf510c0d97R2486